skrifa/src/outline/mod.rs

```diff
@@ -585,6 +587,21 @@ impl<'a> OutlineGlyphCollection<'a> {
         })
     }
 
+    /// Returns true if the interpreter engine should be used for hinting this
+    /// set of outlines.
+    ///
+    /// When this returns false, you likely want to use the automatic hinter
+    /// instead.
+    ///
+    /// This matches the logic used in FreeType when neither of the
+    /// `FT_LOAD_FORCE_AUTOHINT` or `FT_LOAD_NO_AUTOHINT` flags are specified.
+    pub fn prefer_interpreter(&self) -> bool {
```

Comment:
Perhaps explain in this comment from where this is used and how it relates to the `Engine` configuration option?

Reply:
Will do.

Reply:
And this one too